plugins/kubernetes-backend/src/service/KubernetesFanOutHandler.ts

```diff
@@ -26,38 +26,38 @@ import { KubernetesRequestBody } from '@backstage/plugin-kubernetes-common';
 import { KubernetesAuthTranslator } from '../kubernetes-auth-translator/types';
 import { KubernetesAuthTranslatorGenerator } from '../kubernetes-auth-translator/KubernetesAuthTranslatorGenerator';
 
-const DEFAULT_OBJECTS = new Set<KubernetesObjectTypes>([
+const DEFAULT_OBJECTS: KubernetesObjectTypes[] = [
```

Comment:
Think we should export this too in order for the API to grow.

I wonder if it also makes sense to make this constructor an object rather than single args?

Reply:
Thanks, yea I agree with both!